test/integration/dredd-test.coffee

```diff
@@ -158,10 +158,11 @@ describe "Dredd class Integration", () ->
       server = app.listen PORT, () ->
         server2 = apiary.listen (PORT+1), ->
 
-          #Comment out this timeout to enable race condition bug
+          # Comment out this timeout to enable race condition bug
+          # Event loop get stuck or idle without this
```

Comment:
I think this comment is useless to future generations without URL to an actuall issue on GitHub with description of the bug.